Gazebo UR installation

### Affected ROS2 Driver version(s)

Gazebo with Humble

### Used ROS distribution.

Humble

### Which combination of platform is the ROS driver running on.

Ubuntu Linux with standard kernel

### How is the UR ROS2 Driver installed.

Build both the ROS driver and UR Client Library from source

### Which robot platform is the driver connected to.

URSim in docker

### Robot SW / URSim version(s)

ursim polycope X 10.9

### How is the ROS driver used.

Headless without using the teach pendant

### Issue details

# Summary
*Short introduction to the issue and how it impact you and why*

I tried to install  gazebo like mentioned [here](https://docs.universal-robots.com/Universal_Robots_ROS_Documentation/doc/ur_simulation_gz/ur_simulation_gz/doc/installation.html)
## Issue details
*Detailed description help us understand the problem. Code are welcome!*

Everything works until you execute: 

         colcon build --symlink-install

Everything is sourced on my site. 

 
## Steps to Reproduce
*Make simple example to reproduce the issue. Try to remove dependencies to other hardware and software components, if it is possible.*

Execude all commands on the [page](https://docs.universal-robots.com/Universal_Robots_ROS_Documentation/doc/ur_simulation_gz/ur_simulation_gz/doc/installation.html). 
## Expected Behavior
*What did you expect and why?*

## Actual Behavior
*What did you observe? If possible please attach relevant information.*

# Workaround Suggestion
*If a workaround has been found, you are welcome to share it.*


### Relevant log output

        ```shell
        colcon build --symlink-install
        Starting >>> ur_dashboard_msgs
        Starting >>> ur_moveit_config
        --- stderr: ur_moveit_config
        failed to create symbolic link '/home/aut-laptop/ur_ws/build/ur_moveit_config/ament_cmake_python/ur_moveit_config/ur_moveit_config' because existing path cannot be removed: Is a directory
        gmake[2]: *** [CMakeFiles/ament_cmake_python_symlink_ur_moveit_config.dir/build.make:70: CMakeFiles/ament_cmake_python_symlink_ur_moveit_config] Fehler 1
        gmake[1]: *** [CMakeFiles/Makefile2:138: CMakeFiles/ament_cmake_python_symlink_ur_moveit_config.dir/all] Fehler 2
        gmake: *** [Makefile:146: all] Fehler 2
        ---
        Failed   <<< ur_moveit_config [0.06s, exited with code 2]
        Aborted  <<< ur_dashboard_msgs [0.07s]
        
        Summary: 0 packages finished [0.27s]
          1 package failed: ur_moveit_config
          1 package aborted: ur_dashboard_msgs
          2 packages had stderr output: ur_dashboard_msgs ur_moveit_config
          6 packages not processed
        ```

### Accept Public visibility

- [x] I agree to make this context public

--- a/ur_simulation_gz/doc/installation.rst
+++ b/ur_simulation_gz/doc/installation.rst
@@ -3,10 +3,44 @@
 Installation
 ============
 
-Skip any of below steps is not applicable.
+You can install the Universal Robots ROS2 GZ Simulation package in two ways:
+
+1. **From binary packages**: This is the recommended way for most users, as it is quick and easy.
+2. **From source**: This is useful if you want to modify the code or if you require to have a
+   version that is newer than the latest binary package available.
+
+Installation from binary packages
+---------------------------------
+
+1. `Install ROS2 <https://docs.ros.org/en/rolling/Installation/Ubuntu-Install-Debians.html>`_
+2. Install the package using
+
+.. code-block:: bash
+
+  sudo apt-get install ros-${ROS_DISTRO}-ur-simulation-gz
+
+That's it! You can now use the Universal Robots ROS2 GZ Simulation package in your ROS 2
+environment. See the :ref:`ur_simulation_gz_usage` section for how to use it.
+
+
+Installation from source
+------------------------
+
+.. note::
+   This section is only required if you want to build the package from source. Please do that
+   only, if that's required for you. Reasons why you would need to build it from source are:
+
+   - You built one of multiple of this package's dependencies from source and want to use it with
+     this package.
+   - You want to develop this package or contribute to it. If you only want to modify the launch
+     files to fit your application, a better approeach would be to create your own package with
+     your own launch files that include the ones from this package.
+
+   To rather install the binary packages, please refer to the `Installation from binary
+   packages`_ section above.
 
 Setup ROS2 Workspace
---------------------
+~~~~~~~~~~~~~~~~~~~~
 
 1. Create a colcon workspace:
 
@@ -30,11 +64,10 @@ Setup ROS2 Workspace
 
     $ cd $COLCON_WS
     $ git clone -b ros2 https://github.com/UniversalRobots/Universal_Robots_ROS2_GZ_Simulation.git src/ur_simulation_gz
-    $ vcs import --input src/ur_simulation_gz/ur_simulation_gz-not-released.rolling.repos src # only required for rolling
     $ rosdep update && rosdep install --ignore-src --from-paths src -y
 
 Configure and build Workspace
------------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 To configure and build the workspace execute following commands:
 

--- a/ur_simulation_gz/doc/usage.rst
+++ b/ur_simulation_gz/doc/usage.rst
@@ -1,5 +1,7 @@
 :github_url: https://github.com/UniversalRobots/Universal_Robots_ROS2_GZ_Simulation/blob/ros2/ur_simulation_gz/doc/usage.rst
 
+.. _ur_simulation_gz_usage:
+
 Usage
 =====
 

